cf-keycrypt/cf-keycrypt.c

```diff
@@ -0,0 +1,388 @@
+/*
+   cf-keycrypt.c
+
+   Copyright (C) 2017 cfengineers.net
```

Comment:
I think we should add our copyright notice as well. Maybe also convert the language to GPLv3. Pinging @kacf and @nickanderson. 

We should definitely mention the primary author of this in the CONTRIBUTORS file, and probably ping him in this pull request, what's his handle?